bluebottle/bb_projects/models.py

```diff
@@ -236,6 +235,10 @@ def people_needed(self):
 
         return requested - counts['total'] + (counts['externals'] or 0)
 
+    @property
+    def account_bic(self):
+        return self.account_details
+
```

Comment:
I guess this is here because of backwards compatibility, but this isn't writeable. Do you think we still have the desired behaviour? @ashutoshb